docs/style.en-US.md

```diff
@@ -0,0 +1,180 @@
+---
+order: 6
+title: Style
+type: Introduction
+---
+
+This document is mainly about how to use and organize style files in projects. If you want to get a basic knowledge of CSS or look for properties usage, you can refer to the [MDN doc] (https://developer.mozilla.org/en-US/docs/Web/CSS/Reference).
+
+---
+
+## less
+
+Ant Design Pro defaults to using less as the style language, we recommended that you learn about the features of [less](http://lesscss.org/) before using it or sometimes when you have some questions.
+
+## CSS Modules
+
+In the style development process, there are two problems are prominent:
+
+- Global Pollution - CSS file selectors are globally valid. according to the build file generated in the order, the ahead selector's style will be cover by the back one which has the same selector name;
+- Selector complex - in order to avoid the above problem, we have to be careful when writing styles. the class name will be brought flag for limit the style's scope lead to it will getting longer and longer. And in multi-person development, it not only easily lead to the naming style confusion but also cause the number of selectors used on an element be more and more.
+
+In order to solve the above problems, our scaffold use CSS Modules as a modular solution. let us have a look at how to write style in this mode.
+
+```html
+// example.js
+import styles from './example.less';
+
+export default ({title}) => <div className={styles.title}>{title}</div>;
+```
+
+```css
+// example.less
+.title {
+  color: @heading-color;
+  font-weight: 600;
+  margin-bottom: 16px;
+}
+```
+
+Write style use less file does not seem to change, but the class name is relatively simple (the actual project is also the case). js file change at the className property, it changed with an object attribute instead of the original string. the object is import from the less file, and the attribute name same as the selector name in less file.
```

Comment:
Write style use less file does not seem to change `much`

Reply:
...The classname in js files would be replaced by an object attribute, which has the same name as the selector in the less file from where the object was imported.